src/api/status/public/style.css

```diff
@@ -1,5 +1,8 @@
-#results {
-  max-width: 600px;
+#sidenav-collapse-main > ul > li > a.active {
+  background-color: #121d59;
```

Comment:
This file should be deleted.

Reply:
@Andrewnt219 As I mentioned above... it needs... please read the comments above...

Reply:
This file is a duplicate of site.css.

Reply:
deleted